test/test_helper.rb

```diff
@@ -25,6 +25,8 @@
 require './test/support/classes'
 require './test/support/helpers'
 
+require "rails/engine/railties"
+require "sidekiq/rails"
```

Comment:
Could you please move these 2 `require`s some lines above? It seems more consistent to load external deps in the same place.